Test.Bot.Core/TestNightPhase.cs

```diff
@@ -100,12 +100,20 @@ public void TwoStorytellers_BothMoveToSameCottage()
 
             TownSquareMock.SetupGet(t => t.Users).Returns(new[] { InteractionAuthorMock.Object, st2.Object, Villager1Mock.Object });
 
-            var gameMock = CreateGameMock();
-            gameMock.SetupGet(g => g.Storytellers).Returns(new[] { InteractionAuthorMock.Object, st2.Object });
-            gameMock.SetupGet(g => g.Villagers).Returns(new[] { Villager1Mock.Object });
+            MockGameInProgress();
+            // Add a second storyteller
+            UserShouldHaveRole(st2, StorytellerRoleMock.Object);
 
             BotGameplay g = new(GetServiceProvider());
-            AssertCompletedTask(() => g.PhaseNightUnsafe(gameMock.Object, ProcessLoggerMock.Object));
+            
+            // Get the Game
+            var gameTask = g.CurrentGameAsync(MockTownKey, InteractionAuthorMock.Object, new ProcessLogger());
```

Comment:
Can simplify with wrapper:
```
var game = AssertCompletedTask(() => g.CurrentGameAsync(etc.));
Assert.NotNull(game);
```

Reply:
Why is this not using `ProcessLoggerMock.Object`?

Reply:
I have no idea why I didn't use the ProcessLoggerMock. I plead "it was midnight"

Reply:
And somehow I totally missed the generic version of AssertCompletedTask, wow. I looked for something like this, thought i didn't exist, and didn't want to hassle with making it :)